osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -22,6 +23,12 @@ class ConversionApiClient(RESTApiJWTClient):
     conversion_job_url = '/jobs/'
     conversion_job_status_url = '/conversion_result/{job_uuid}/'
 
+    @staticmethod
+    def _extraction_processing_overdated(progress, extraction_order):
+        return (progress in ['new', 'received', 'started']) and timezone.now() > (
+            extraction_order.process_start_time + settings.OSMAXX.get('EXTRACTION_PROCESSING_TIMEOUT_TIMEDELTA')
```

Comment:
`process_start_time` can be explicitly null (None), which gives an error whenever adding with `timedelta`. Maybe checking if `extraction_order.process_start_time` is not None?